telecine/src/main/AndroidManifest.xml

```diff
@@ -47,6 +47,16 @@
         android:taskAffinity=""
         android:exported="true"
         />
+
+    <service
+        android:icon="@drawable/ic_videocam_white_24dp"
+        android:label="@string/app_name"
```

Comment:
Yeah I was originally planning on using a variation of the launcher icon but you already had this one for notifications so I went with that. The shortcut just uses "Launch". Is the icon enough context to indicate it's Telecine?